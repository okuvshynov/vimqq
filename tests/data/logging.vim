let g:vqq_log_file = "logging.out"
let g:vqq_log_format = "%H:%M "
call vimqq#log#info("hello world")
:qa!
