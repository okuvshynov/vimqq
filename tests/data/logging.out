I00:00 hello world
